### DISCUSSION TITLE
How do I prevent the automatic? <C-E> mapping?

### DISCUSSION THREAD

**Discussion by galou** [hrsh7th/nvim-cmp#854] [Q&A] [answered]
On my configuration, a mapping to `<C-E>` in insert mode is set to `<Cmd>call v:lua.cmp.utils.keymap.set_map(18)<CR>` (with a varying number). In my configuration, I did `mapping = nil,` to try to avoid this but it didn't help. Deleting the mapping with `:lua vim.keymap.del({ "i" }, "<C-e>")` doesn't help either because the mapping is created again. How can I prevent this?

**Comment by zer09** [comment]
Maybe use ```inoremap <C-e>``` for the mapping

**Comment by hrsh7th** [comment, marked as answer]
You can disable mapping via `cmp.config.disable`

```lua
cmp.setup {
  mapping = {
    ...
    ['<C-e>'] = cmp.config.disable
    ...
  }
}
```

**Reply by galou** [reply]
Thanks! That was it.
